### PR TITLE
Update project organization

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description  | Update project organization with new members
| Fixed ticket |

### CODE REVIEW

File: content/maintainers-guide/project-organization.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Current product managers:
 
 - Fabien Vallon ([@fabienvallon](https://github.com/fabienvallon))
 - Mateus Shirlaw ([@MatShir](https://github.com/MatShir))
+- Eshane Rawat ([@eshraw](https://github.com/eshraw))
```
Comment (matks) [inline content/maintainers-guide/project-organization.md:18]: If you can confirm @MatShir that is is official

File: content/maintainers-guide/project-organization.md [hunk 2]
```diff
@@ -108,6 +110,7 @@ Wording managers ensure that wordings are adequate, follow the project's tone of
 Current wording managers:
 
 - Julie Varisellaz ([@Julievrz](https://github.com/Julievrz))
+- Léa Delin ([@l-delin](https://github.com/l-delin))
```
Comment (matks) [inline content/maintainers-guide/project-organization.md:113]: If you can confirm @Julievrz that is is official

File: content/maintainers-guide/project-organization.md [hunk 3]
```diff
@@ -116,6 +119,7 @@ User documentation managers ensure the quality and consistency of the [user docu
 Current user documentation managers:
 
 - Julie Varisellaz ([@Julievrz](https://github.com/Julievrz))
+- Léa Delin ([@l-delin](https://github.com/l-delin))
```
Comment (matks) [inline content/maintainers-guide/project-organization.md:122]: If you can confirm @Julievrz that is is official

File: content/maintainers-guide/project-organization.md [hunk 4]
```diff
@@ -52,6 +53,7 @@ Current committers:
 - Marco Salvatore ([@marsaldev](https://github.com/marsaldev))
 - Thomas Roux ([@okom3pom](https://github.com/okom3pom))
 - Tomasz Zając ([@SharakPL](https://github.com/SharakPL))
+- Fabien Papet ([@FabienPapet](https://github.com/FabienPapet))
```
Comment (matks) [inline content/maintainers-guide/project-organization.md:56]: Consequence of https://github.com/PrestaShop/open-source/issues/121

**Review by kpodemski — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by PrestaEdit** [comment]
Can we add also that somes of them are actually into the PrestaShop SA, maybe ? I think it's usefull to let others know about it.

**Comment by kpodemski** [comment]
@PrestaEdit, I don't think it is necessary. It's true, PrestaShop SA hires Fabien, but he needs to be a committer first to have a chance to become a maintainer. Not sure how this information could be helpful to others. Everyone is trying their best, no matter their employment status.

**Comment by PrestaEdit** [comment]
It's not about @Fabien, it's about everyone.
As PrestaShop SA hire some of them, I think it could be usefull to open-source contributors to get this specific information, about this fact.

**Comment by kpodemski** [comment]
I'm not sure how could that be useful ;-)

We can ping @eternoendless so that he can decide.

**Comment by PrestaEdit** [comment]
Or maybe ping someone other than a PrestaShop SA employee ? :trollface:

**Pull request merged**